bundles/io/org.eclipse.smarthome.io.rest.sitemap/OSGI-INF/sitemapsubscriptionservice.xml

```diff
@@ -15,4 +15,5 @@
    <service>
       <provide interface="org.eclipse.smarthome.io.rest.sitemap.SitemapSubscriptionService"/>
    </service>
+   <reference bind="addModelRepository" cardinality="0..n" interface="org.eclipse.smarthome.model.core.ModelRepository" name="ModelRepository" policy="dynamic" unbind="removeModelRepository"/>
```

Comment:
`cardinality="0..1"` - there will only be one (max), and btw. your implementation also is not prepared for more than that